<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pdfWidget</class>
 <widget class="QWidget" name="pdfWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简洁PDF</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Data/SimplePdf.qrc">
    <normaloff>:/SimplePdf/res/title.png</normaloff>:/SimplePdf/res/title.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../Data/SimplePdf.qrc">
        <normaloff>:/SimplePdf/res/split.png</normaloff>:/SimplePdf/res/split.png</iconset>
      </attribute>
      <attribute name="title">
       <string>分割</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>文件：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_Total">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="placeholderText">
          <string>请输入页数范围，如：0-100,150-200,150-300</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_split">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>80</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton{font: 28pt &quot;微软雅黑&quot;;border-radius: 8px;
color: rgb(255, 255, 255);background-color: rgb(229, 50, 45);}
QPushButton:hover{font: 28pt &quot;微软雅黑&quot;;border-radius: 8px;
color: rgb(255, 255, 255);background-color: rgb(22, 22, 22);}
QPushButton:pressed{font: 28pt &quot;微软雅黑&quot;;border-radius: 8px;
color: rgb(255, 255, 255);background-color: rgb(229, 50, 45);}</string>
           </property>
           <property name="text">
            <string>拆分PDF</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../Data/SimplePdf.qrc">
        <normaloff>:/SimplePdf/res/merge.png</normaloff>:/SimplePdf/res/merge.png</iconset>
      </attribute>
      <attribute name="title">
       <string>合并</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_merge">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>80</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton{font: 28pt &quot;微软雅黑&quot;;border-radius: 8px;
color: rgb(255, 255, 255);background-color: rgb(229, 50, 45);}
QPushButton:hover{font: 28pt &quot;微软雅黑&quot;;border-radius: 8px;
color: rgb(255, 255, 255);background-color: rgb(22, 22, 22);}
QPushButton:pressed{font: 28pt &quot;微软雅黑&quot;;border-radius: 8px;
color: rgb(255, 255, 255);background-color: rgb(229, 50, 45);}</string>
           </property>
           <property name="text">
            <string>合并PDF</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>HTML转PDF</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="listWidget_html"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_merge_html">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>80</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton{font: 28pt &quot;微软雅黑&quot;;border-radius: 8px;
color: rgb(255, 255, 255);background-color: rgb(229, 50, 45);}
QPushButton:hover{font: 28pt &quot;微软雅黑&quot;;border-radius: 8px;
color: rgb(255, 255, 255);background-color: rgb(22, 22, 22);}
QPushButton:pressed{font: 28pt &quot;微软雅黑&quot;;border-radius: 8px;
color: rgb(255, 255, 255);background-color: rgb(229, 50, 45);}</string>
           </property>
           <property name="text">
            <string>HTML转PDF</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Data/SimplePdf.qrc"/>
 </resources>
 <connections/>
</ui>
